android/src/test/java/com/revenuecat/purchases/hybridcommon/mappers/StoreProductMapperTests.kt

```diff
@@ -105,6 +105,19 @@ internal class StoreProductMapperTest {
             assertThat(it["productType"]).isEqualTo("UNKNOWN")
         }
     }
+
+    @Test
+    fun `maps subscription period correctly`() {
+        stubStoreProduct().map().let {
+            assertThat(it["subscriptionPeriod"]).isNull()
+        }
+        stubStoreProduct(subscriptionPeriod = "P1M").map().let {
+            assertThat(it["subscriptionPeriod"]).isEqualTo("P1M")
+        }
+        stubStoreProduct(subscriptionPeriod = "P1Y").map().let {
+            assertThat(it["subscriptionPeriod"]).isEqualTo("P1Y")
```

Comment:
Nice thanks! It could also be interesting to add a test that checks the number of keys on the map so we detect if something changed.